src/bridge/mod.rs

```diff
@@ -109,6 +109,16 @@ fn build_nvim_cmd_with_args(bin: &str) -> Command {
         cmd.args(&["$SHELL", "-lc", &argstring]);
         return cmd;
     }
+
+    #[cfg(macos)]
```

Comment:
Is this clippy error, something we need to be concerned of?

Reply:
I'll look into it